test/lib/BedrockTester.cpp

```diff
@@ -331,35 +275,35 @@ vector<pair<string,string>> BedrockTester::executeWaitMultiple(vector<SData> req
                         }
                     } else {
                         timeouts++;
-                        if (timeouts > 5) {
-                            //SAUTOLOCK(listLock);
-                            //cout << "Timeout (" << timeouts << ") waiting on socket, will try again." << endl;
-                        }
-                        if (timeouts == 60) {
+                        if (timeouts == 600) {
```

Comment:
You changed this to `600` but use `60` below.  Is there a reason for that?

Reply:
I updated the other one to match.